pkgs/development/python-modules/crccheck/default.nix

```diff
@@ -1,18 +1,21 @@
-{ lib, stdenv, buildPythonPackage, fetchPypi
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy3k
 , nose }:
 
-buildPythonPackage rec {
+let
   pname = "crccheck";
-  version = "0.6";
+  version = "1.0";
```

Comment:
I would prefer if you wouldn't use let here but if you want to keep it is fine for me.

Reply:
And I prefer to use let blocks to avoid recursive attribute sets because they increase the scope complexity.

Reply:
All python packages by the core python maintainer team use rec so I wouldn't mind if you did, too but whatever.